include/OpenSpaceToolkit/Astrodynamics/Trajectory/Propagator.hpp

```diff
@@ -130,28 +174,11 @@ class Propagator
 
     void print(std::ostream& anOutputStream, bool displayDecorator = true) const;
 
-    /// @brief              Create a medium fidelity Propagator object with recommended settings
-    ///
-    /// @code
-    ///                     Propagator propagator = Propagator::MediumFidelity(aState) ;
-    /// @endcode
-    /// @param              [in] aState A State
-    /// @return             Propagator
-
-    static Propagator MediumFidelity();
-
-    /// @brief              Create a high fidelity Propagator object with recommended settings
-    ///
-    /// @code
-    ///                     Propagator propagator = Propagator::HighFidelity(aState) ;
-    /// @endcode
-    /// @param              [in] aState A State
-    /// @return             Propagator
-
-    static Propagator HighFidelity();
+    /// @brief              Undefined
+    static Propagator Undefined();
 
    private:
-    mutable SatelliteDynamics satelliteDynamics_;
+    Array<Shared<Dynamics>> dynamics_ = Array<Shared<Dynamics>>::Empty();
```

Comment:
@vishwa2710 So I had removed the default at that level, following-up on that thread on good practices [here](https://stackoverflow.com/questions/11594846/default-member-values-best-practice). Do we actually need to define it here or can we have it at constructor level only? 👀

Reply:
Ah shit my bad. That makes sense thanks for sharing the link.

Reply:
Ah shit my bad. It can be constructor only for sure.